presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3812,17 +3813,17 @@ public void testFileModifiedTimeHiddenColumn()
         assertEquals(getPartitions("test_file_modified_time").size(), 3);
 
         MaterializedResult results = computeActual(format("SELECT *, \"%s\" FROM test_file_modified_time", FILE_MODIFIED_TIME_COLUMN_NAME));
-        Map<Integer, Long> fileModifiedTimeMap = new HashMap<>();
+        Map<Integer, ZonedDateTime> fileModifiedTimeMap = new HashMap<>();
```

Comment:
ZonedDateTime -> Instant

(file modification times are points in time, zoneless)